test/Microsoft.ComponentDetection.VerificationTests/Microsoft.ComponentDetection.VerificationTests.csproj

```
@@ -18,7 +18,8 @@
         IF TESTING LOCALLY, UNCOMMENT ABOVE AND COMMENT BELOW -->
         <PackageReference Include="FluentAssertions" />
         <PackageReference Include="Microsoft.Extensions.Logging" />
-        <PackageReference Include="MSTest" />
+        <PackageReference Include="MSTest.TestAdapter "/>
```

This is missing some packages:

```suggestion
        <PackageReference Include="MSTest.Analyzers"/>
        <PackageReference Include="MSTest.TestAdapter "/>
        <PackageReference Include="MSTest.TestFramework"/>
```

---

we get that package and `MSTest.TestFramework` from https://github.com/microsoft/component-detection/blob/main/test/Directory.Build.props#L6-L7 , removing `MSTest.TestFramework` from this csproj